developer/src/server/src/site/test.js

```diff
@@ -238,11 +238,10 @@ window.onload = function() {
   keyman.addEventListener('keyboardchange', function(keyboardProperties) {
     if(newOSK) {
       keyman.osk = newOSK;
-      newOSK.activeKeyboard = keyman.core.activeKeyboard;
+      newOSK.activeKeyboard = keyman.contextManager.activeKeyboard;
```

Comment:
```suggestion
      newOSK.activeKeyboard = keyman.contextManager.activeKeyboard;  // Private API refs on both sides
```

Are these still internal references? I believe `keyman.core.activeKeyboard` was not a published API?

Reply:
I suppose they are.  That said, `newOSK.activeKeyboard` or something similar should eventually become "public" for a published, independent OSK module, though possibly after a proper API redesign phase.